profiling/perfetto-connector/perfetto/perfetto.cc

```diff
@@ -72862,7 +72862,7 @@ void ProducerIPCClientImpl::OnConnect() {
 
   // If there are pending Sync() requests, send them now.
   for (const auto& pending_sync : pending_sync_reqs_)
-    Sync(std::move(pending_sync));
+    Sync(pending_sync);
```

Comment:
I would have dropped the `const`-qualifier in the range-based for loop init-statement instead.

I don't think it matters too much but I checked how upstream perfetto resolved this and they did exactly that.
https://github.com/google/perfetto/blob/34953f0ef39e5496fcf9f4dd34e67de6b8dd100f/src/tracing/ipc/producer/producer_ipc_client_impl.cc#L233-L234